dts/arm/renesas/gen3/r8a77951.dtsi

```diff
@@ -5,3 +5,36 @@
  */
 
 #include <renesas/gen3/rcar_gen3_cr7.dtsi>
+#include <dt-bindings/pinctrl/rcar-pinctrl.h>
+
+&pfc {
```

Comment:
>> This driver and dts bindings differs from the Linux one
>
> I'm not referring to the implementation, but how things are described in Devicetree.

Regarding this, please see the general upstream binding rules we recently added to the documentation:

https://docs.zephyrproject.org/latest/guides/dts/bindings.html#check-for-existing-bindings

In particular, see this requirement:

> , if there is an existing binding for your device in an authoritative location, you should try to replicate its properties when writing a Zephyr binding, and you must justify any Zephyr-specific divergences.

I'd like to see the divergence justified. I understand the complexity concerns in source code, but can the existing sources be implemented in terms of a subset of the upstream binding? If not, why not?

Reply:
Hi thanks for your comments.
I implemented pull-up and pull-down features, as well as drive strength settings.
I'm still fare from the Linux upstream binding, I tried to explain the reasons and the difficulties.
Since I did a lot of modifications to the driver and the binding I created a new pull request #40550 which
supersedes this one.